server/application-server/src/main/resources/application-prod.yml

```
@@ -30,6 +47,13 @@
     intelligence-service:
         url: ${INTELLIGENCE_SERVICE_URL}
 
+    mail:
+      enabled: ${MAIL_ENABLED:true}
+      sender: ${MAIL_SENDER:hephaestus@aet.cit.tum.de}
+
+    detection:
+        automatic-detection-enabled: ${AUTOMATIC_DETECTION_ENABLED:false}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)